sdks/java/io/snowflake/src/main/java/org/apache/beam/sdk/io/snowflake/xlang/package-info.java

```diff
@@ -0,0 +1,2 @@
+/** Cross-language for SnowflakeIO. */
+package org.apache.beam.sdk.io.snowflake.xlang;
```

Comment:
Add annotations:
@Experimental(Kind.PORTABILITY)
@DefaultAnnotation(NonNull.class)

Reply:
why to add @DefaultAnnotation(NonNull.class) ?

Reply:
It's for spotbugs, I was recommended to add it in my code

Reply:
okie